packages/scrum-client/src/config/routes.js

```diff
@@ -8,85 +8,106 @@ import VotingCards from '../content/VotingCards';
 import PrivacyPolicy from '../content/PrivacyPolicy';
 import NotFound from '../content/NotFound';
 import PATHS from '../constants/paths';
+import PATH_PARAMS from '../constants/pathsParams';
 import ShareSession from '../content/ShareSession/ShareSession';
 import ParticipateSession from '../content/ParticipateSession/ParticipateSession';
 import OauthValidation from '../content/OauthValidation/OauthValidation';
+import Profile from '../content/Profile';
 
 const routes = [
   {
     path: PATHS.LOGIN,
     component: Login,
     title: 'Login',
+    params: PATH_PARAMS.NO_PARAM,
```

Comment:
chavito , if no params is added why is it needed in the first place the property ? 
It would not be better to avoid assigning `params`?

Reply:
You are probably right. Let me look into it